src/test/java/com/connexta/store/controllers/StoreControllerTest.java

```diff
@@ -39,44 +34,10 @@ public void beforeEach() {
     storeApi = new StoreController(mockStoreService, STORE_API_VERSION);
   }
 
-  @ParameterizedTest
-  @NullAndEmptySource
-  @ValueSource(strings = "   ")
-  public void testBadContentType(
-      final String contentType, @Mock final MultipartFile mockMultipartFile) {
-    when(mockMultipartFile.getContentType()).thenReturn(contentType);
-    assertBadRequest(mockMultipartFile);
-  }
-
-  @ParameterizedTest
-  @NullAndEmptySource
-  @ValueSource(strings = "   ")
-  public void testBadFilename(final String filename, @Mock final MultipartFile mockMultipartFile) {
-    when(mockMultipartFile.getContentType()).thenReturn("testContentType");
-    when(mockMultipartFile.getOriginalFilename()).thenReturn(filename);
-    assertBadRequest(mockMultipartFile);
-  }
-
   @Test
-  void testFileTooLarge(@Mock final MultipartFile mockMultipartFile) {
```

Comment:
Why were these removed? These tests are still valid for the ingest endpoint.

Reply:
temporary solution only

Reply:
I put the same checks in the storecontrolleringesttest